docs/visual-basic/reference/vb6-support.md

```diff
@@ -14,21 +14,21 @@ ms.assetid: ffc5ba4d-44d7-4ef7-a3f6-38a8738bf127
 author: "paulyuk"
 ms.author: "paulyuk"
 ---
-# Support Statement for Visual Basic 6.0 on Windows Vista, Windows Server 2008, Windows 7, Windows 8 and Windows 8.1, Windows Server 2012, Windows 10, and Windows Server 2016
+# Support Statement for Visual Basic 6.0 on Windows
 
-Update: March 2016
```

Comment:
Is it necessary to specify the update date twice? Currently, the start of the article looks like this:

![](https://user-images.githubusercontent.com/287848/29810375-d749fe20-8c9f-11e7-8c8f-ac67f8c0bc56.png)

Reply:
No, we should remove this line

Reply:
Thanks @svick @mairaw!  Removed in latest commit to the PR.